<launch>

  <node name="cameraIR_arm" pkg="bit_hardware_interface" type="interface_camerair" output="screen" respawn="false">
    <param name="DeviceAddress"       value="192.168.50.11"/>  
    <param name="Port"   value="8000"/>
    <param name="UserName"   value="admin"/>
    <param name="Password"   value="space305"/>
    <param name="PictureName"   value="./src/bit_hardware_interface/src/camerair_arm.jpg"/>
    <param name="MsgName"   value="imagearm"/>
  </node>

  <node name="cameraIR_car" pkg="bit_hardware_interface" type="interface_camerair" output="screen" respawn="false">
    <param name="DeviceAddress"       value="192.168.50.12"/>  
    <param name="Port"   value="8000"/>
    <param name="UserName"   value="admin"/>
    <param name="Password"   value="space305"/>
    <param name="PictureName"   value="./src/bit_hardware_interface/src/camerair_car.jpg"/>
    <param name="MsgName"   value="imagecar"/>
  </node>

</launch>



